### ISSUE TITLE
Add 'P or old variables to req-format

### ISSUE THREAD

**Issue by danieldietsch** [ultimate-pa/ultimate#463]
* Each variable gets an old-variant accessible via the old-expression
* In the transformation, we get an explicit old variable for each variable. 
* After each transition, we update old variables.

**Comment by jgarciaDevelopair** [comment]
The semantic part is done, it is still missing the front part

**Comment by hauff** [comment]
Implementation:
- [x] Create or find some CDDWalker that substitutes `prev(R)` by `'R`.
- [x] Add copy constructor to each pattern class.
- [x] Transform CDDs in Req2Pea before `constructSymbolTable()`.

**Comment by alopez-developair** [comment]
We are having trouble making use of this feature.

We tried this:

```
Input A is bool
Input B is bool

R1: Globally, it is always the case that if "A" holds, then "B == !prev(B)" holds after at most "5" time units.
```

and the same by changing the requirement to

```
R1: Globally, it is always the case that if "A" holds, then "B == !'B" holds after at most "5" time units.
```

In both cases, the output we get is the following:

```
--- Results ---
 * Results from de.uni_freiburg.informatik.ultimate.pea2boogie:
  - ExceptionOrErrorResult: AssertionError: 'B' has no identifier expression or no type
    de.uni_freiburg.informatik.ultimate.pea2boogie: AssertionError: 'B' has no identifier expression or no type: de.uni_freiburg.informatik.ultimate.pea2boogie.translator.ReqSymboltableBuilder$ReqSymbolTable.get
IdentifierExpression(ReqSymboltableBuilder.java:473)
```

Just to be sure I tried out the only example I found in the repo (https://github.com/ultimate-pa/ultimate/blob/211abdf072b9b2fa405c68cf514740f8d2197197/trunk/examples/Requirements/prev_function_application.req) and I got something similar.

Thoughts? Maybe we have to re-open this issue?

**Comment by danieldietsch** [comment]
What version are you using?

**Comment by alopez-developair** [comment]
> What version are you using?

0.1.25-d7dd348

(should be one of the last ones before the transition to Java 11)

**Comment by danieldietsch** [comment]
Ill look into it

**Comment by henkele** [comment]
Should be solved by 6d31cd06afbc782544c2a13988275ddc91bd1fe6
I checked using the above requirement, yields no exception.
